firmware/application/ui_record_view.cpp

```diff
@@ -113,7 +113,7 @@ uint32_t RecordView::set_sampling_rate(uint32_t new_sampling_rate) {
      * They are ok as recorded spectrum indication, but they should not be used by Replay app. (the voice speed will be accelerated)
 
      * We keep original black background in all the correct IQ .C16 files BW's Options. */
-    if (actual_sampling_rate > 8'000'000) {  // Till fw 1.7.4  BW limit yellow REC button  was >500khz, now  BW >1Mhz  (fs=8*BW), (>1Mhz ..2M75)
+    if ((actual_sampling_rate > 8'000'000) || (actual_sampling_rate == 1'280'000) || (actual_sampling_rate == 640'000)) {  // yellow REC button means not ok for REC, BW >1Mhz , BW < 150khz
```

Comment:
I would propose to eliminate the two =='s to match the comments, perhaps:
  `if ((actual_sampling_rate > 8'000'000) || (actual_sampling_rate <= 1'280'000))`